style.css

```diff
@@ -330,7 +328,118 @@ li {
   color: #f55800;
 }
 
-.button:disabled {
-  background-color: #f4f1f0;
-  color: #5b5957;
+.form {
+  height: 844px;
+  background-color: #fff;
+  display: flex;
+  min-width: 375px;
+  flex-direction: column;
+  gap: 58px;
+  padding: 120px 24px 120px 24px;
+  justify-content: center;
+}
+
+.form-text {
+  font-family: 'Inter', sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  color: #172b4d;
+  line-height: 44px;
+  text-align: center;
+  width: 327px;
+  height: 216px;
+  margin: auto;
+}
+
+.form-form {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+  align-items: center;
+  height: 330px;
+  margin: auto;
+  margin-top: 0;
+}
+
+.input {
+  display: flex;
+  flex-direction: column;
+  gap: 14px;
+  height: 48px;
+  width: 327px;
+  margin: 0;
+}
+
+.input-x {
+  height: 18px;
+  width: 295px;
+  border: none;
+  padding-left: 16px;
+  padding-right: 16px;
+}
+
+input::placeholder {
+  color: #979493;
+}
+
+.textarea {
+  font-family: 'Inter', sans-serif;
+  font-size: 15px;
+  height: 114px;
+  width: 100%;
+  background-color: #fbf8f7;
+  margin-top: 24px;
+  padding: 12px;
+  border: none;
+}
+
+textarea::placeholder {
+  color: #3c3a39;
+}
+
+.line-y {
+  border: 1px solid #dbd8d7;
+  width: 100%;
+  height: 0;
+}
+
+.button-submit {
+  height: 48px;
+  width: 131px;
+  background-color: #ff6b00;
+  color: #fff;
+  border: none;
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  margin-bottom: 0;
+}
+
+.line-f {
+  border: 1px solid #dfe1e6;
+  width: 100%;
+  padding-bottom: 0;
+}
+
+.line-z {
+  width: 134px;
+  border: 5px solid #0d151e;
+  border-radius: 10px;
+}
+
+.footer {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background-color: #fff;
+  align-items: center;
+  height: 72px;
+}
+
+.social-media-footer {
+  display: flex;
+  gap: 0.5rem;
+  align-items: center;
+  padding-left: 0;
```

Comment:
- [ ] As the previous code reviewer mentioned, there are some styling requirements not met. Please reduce the size of the social links at the bottom and centralize them.
 - Your design:
 - 
<img width="460" alt="image" src="https://user-images.githubusercontent.com/61919617/222140047-0c1e54f9-55d0-43c2-a11b-29d421977943.png">